### DISCUSSION TITLE
Question about Export / unzip / Import

### DISCUSSION THREAD

**Discussion by PhilB-RLB** [TriliumNext/Trilium#2724] [Q&A] [answered]
Hello,

I've just figured out that if I export a note (containing several subnotes) and then import the zip file I obtain an identical note into Trilium.

Now, if I export the same note, unzip the archive then zip the directory and its files and import the new zip archive, the note reimported is not identical. I can see in particular the following 'system files' !!!meta.json / index / navigation and style.css mentioned in the left pane of Trilium after the import.

Why this happened? How to avoid this? (actually I want to export / unzip and slightly modify one file / import).

**Comment by sigaloid** [comment]
The `!!!meta.json` file contains Trilium's metadata files, if it isn't in the root directory of the zip, they get imported as standard files. It sounds like you zipped the upper directory, meaning you zipped the folder containing all the files, rather than zipping the files directly. 

**file.zip**
----↪ :file_folder: files 
--------↪ !!!meta.json
--------↪ other files


Instead, you should create a zip file directly of all of the files, meaning opening the folder containing the files like !!!meta.json, selecting all of them, and _then_ creating the zip file.

**file.zip** 
----↪ !!!meta.json

You can tell if you did this right by double-clicking the zip file and opening it in the zip viewer - if you open the zip file and you have to enter another folder first to see the files, it's not done correctly.

**Comment by PhilB-RLB** [comment, marked as answer]
Thank you sigaloid,
it works now !!!

**Reply by sigaloid** [reply]
Glad to hear :smile:
